### PR TITLE
Improve: bollmaker preloads dynamic spreads

### PR DESCRIPTION
**Pull request by andycheng123**

### CODE REVIEW

**Review by c9s — APPROVED** [review]
go!

**Review by c9s — CHANGES_REQUESTED** [review]
you don't need to load it manually? use standardIndicatorSet?

File: pkg/strategy/bollmaker/dynamic_spread.go [hunk 1]
```diff
@@ -45,6 +44,19 @@ func (ds *DynamicSpreadSettings) Update(kline types.KLine) {
 	}
 }
 
+// Initialize dynamic spreads and preload SMAs
+func (ds *DynamicSpreadSettings) Initialize(symbol string, session *bbgo.ExchangeSession) {
+	ds.DynamicBidSpread = &indicator.SMA{IntervalWindow: types.IntervalWindow{Interval: ds.Interval, Window: ds.Window}}
+	ds.DynamicAskSpread = &indicator.SMA{IntervalWindow: types.IntervalWindow{Interval: ds.Interval, Window: ds.Window}}
```
Comment (c9s) [inline pkg/strategy/bollmaker/dynamic_spread.go:50]: `session.StandardIndicatorSet(s.Symbol).SMA(ds.IntervalWindow)` 這樣應該就可以用了

File: pkg/strategy/bollmaker/dynamic_spread.go [hunk 2]
```diff
@@ -45,6 +44,19 @@ func (ds *DynamicSpreadSettings) Update(kline types.KLine) {
...
+
+	kLineStore, _ := session.MarketDataStore(symbol)
+	if klines, ok := kLineStore.KLinesOfInterval(ds.Interval); ok {
+		for i := 0; i < len(*klines); i++ {
+			ds.Update((*klines)[i])
+		}
+	}
```
Comment (c9s) [inline pkg/strategy/bollmaker/dynamic_spread.go:57]: this part can be removed

**Comment by andycheng123** [comment]
> you don't need to load it manually? use standardIndicatorSet?

it's not kline SMA

File: pkg/strategy/bollmaker/dynamic_spread.go (hunk 1 above)
Comment (andycheng123) [reply pkg/strategy/bollmaker/dynamic_spread.go:50]: it's not kline SMA

Comment (c9s) [reply pkg/strategy/bollmaker/dynamic_spread.go:50]: ohoh!!!

**Review by c9s — APPROVED** [review]

**Pull request closed without merging**